src/components/Video/ThermalVideoPlayer.vue

```diff
@@ -185,10 +185,12 @@ export default {
     },
     bindRateChange() {
       const rate = localStorage.getItem("playbackrate");
+      const htmlPlayer = this.$refs.player.$refs.video;
```

Comment:
It would be nicer if we didn't have to reach so deep through the object heirarchy here. `player` should expose the html player as a property.

I'm ok with you not fixing this now - just commenting.